CHANGELOG.md

```diff
@@ -93,27 +177,27 @@ By @bradwerth [#6216](https://github.com/gfx-rs/wgpu/pull/6216).
 
 #### General
 
-- Add `VideoFrame` to `ExternalImageSource` enum. By @jprochazk in [#6170](https://github.com/gfx-rs/wgpu/pull/6170)
-- Add `wgpu::util::new_instance_with_webgpu_detection` & `wgpu::util::is_browser_webgpu_supported` to make it easier to support WebGPU & WebGL in the same binary. By @wumpf in [#6371](https://github.com/gfx-rs/wgpu/pull/6371)
+- Add `VideoFrame` to `ExternalImageSource` enum. By @jprochazk in [#6170](https://github.com/gfx-rs/wgpu/pull/6170).
+- Add `wgpu::util::new_instance_with_webgpu_detection` & `wgpu::util::is_browser_webgpu_supported` to make it easier to support WebGPU & WebGL in the same binary. By @wumpf in [#6371](https://github.com/gfx-rs/wgpu/pull/6371).
 
 #### Vulkan
 
-- Allow using [VK_GOOGLE_display_timing](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_GOOGLE_display_timing.html) unsafely with the `VULKAN_GOOGLE_DISPLAY_TIMING` feature. By @DJMcNab in [#6149](https://github.com/gfx-rs/wgpu/pull/6149)
+- Allow using [VK_GOOGLE_display_timing](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_GOOGLE_display_timing.html) unsafely with the `VULKAN_GOOGLE_DISPLAY_TIMING` feature. By @DJMcNab in [#6149](https://github.com/gfx-rs/wgpu/pull/6149).
 
 #### Metal
 
-- Implement `atomicCompareExchangeWeak`. By @AsherJingkongChen in [#6265](https://github.com/gfx-rs/wgpu/pull/6265)
+- Implement `atomicCompareExchangeWeak`. By @AsherJingkongChen in [#6265](https://github.com/gfx-rs/wgpu/pull/6265).
```

Comment:
Looks like #6107 is missing here. We ran into this in bevy (see https://github.com/bevyengine/bevy/pull/15988#issuecomment-2439755263). It would be nice to call out needing to add `QuartzCore.framework` to any xcode builds.

Reply:
thanks for calling this out! definitely one that shouldn't be missing 👍 

Btw. note that generally contributors are responsible for adding changelog notes (it's part of the tick boxes in the pr template) and reviewers should point that if missing (which in this case was... me, oops)

Reply:
I'm not sure what content to write for this. @madsmtm and/or @Wumpf, can you please do that? I would also consider fixing this as follow-up, if it comes to that, since this is the only thing that might block merging. Tagging @gfx-rs/wgpu for opinions.

Reply:
I'll just push a commit into this branch if you don't mind

Reply:
Hm doesn't allow to push me. I'd have written something like:

```
- Unless an explicit `CAMetalLayer` is provided, surfaces now render to a sublayer. This improves resizing behavior, fixing glitches during on window resize. By @madsmtm in [#6107](https://github.com/gfx-rs/wgpu/pull/6107).
```

Reply:
Whoopsie, my bad. What @Wumpf has written sounds fine.

Reply:
@Wumpf: That's odd! You should be able to with something like `git push --force git@github.com:erichdongubler-mozilla/wgpu.git HEAD:v23` (N.B., not `--force-with-lease`). 🤔

Pushed myself with a6112742.